src/content/changes/highlights/2022-09-15.md

```diff
@@ -29,3 +29,4 @@ kubectl-gs now also provides a command `get organizations`, which is a way for a
 - [prometheus-remotewrite](https://github.com/giantswarm/prometheus-remotewrite) version [v0.1.0](https://github.com/giantswarm/prometheus-remotewrite/blob/main/CHANGELOG.md#010---2022-07-13) With this release we are enabling Prometheus to replicate its data into 3rd party systems like Grafana Cloud or even another Prometheus using its [remote APIs](https://prometheus.io/blog/2019/10/10/remote-read-meets-streaming/#remote-apis). More details can be found in the [documentation](https://docs.giantswarm.io/ui-api/observability/prometheus-remotewrite/)
 
 ## Documentation
+- We added the [Documentation](https://docs.giantswarm.io/ui-api/observability/grafana/custom-dashboards/) how to package custom dashboards together with apps instead of packaging them into a custom repository. 
```

Comment:
```suggestion
- We added the [Documentation](https://docs.giantswarm.io/ui-api/observability/grafana/custom-dashboards/) on how to package custom dashboards together with apps instead of packaging them into a custom repository. 
```